fabtests/benchmarks/benchmark_shared.c

```diff
@@ -47,6 +47,14 @@
  */
 static int inject_size_set;
 
+/*
+ * When performing RMA with validation, READ needs to ensure it deconflicts
+   it's memory access with the messages sent by ft_sync().  Do this by
+   offsetting all RMA operations away from the beginning of the buffer and
+   leave ft_sync to operate in that region.
+ */
```

Comment:
nit: multi-line comments are usually written as:
```
/* Start writing here, and then
 * continue and
 * continue and
 * continue until done.
 */
```
comment applies to other newly added comments.

Reply:
Fixed.